app/javascript/components/supplemental_application/sections/ConfirmedHouseholdIncome.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react'
+import { find, isNil, uniqBy } from 'lodash'
+
+import FormGrid from '~/components/molecules/FormGrid'
+import FormGroupTextValue from '~/components/atoms/FormGroupTextValue'
+import { RadioGroup, Radio, Text } from 'react-form';
+import { getAMIAction } from '~/components/supplemental_application/actions'
```

Comment:
As a best practice I would call the actions to just in the `Page`. So we push all the state, mutations and actions to the top and can manages the state for the page.

In this practice all components read the state from the parent and modify data thru callbacks to the parent.

We never really talked about this. The application is growing in complexity and it's a good time to talk about patterns/practices etc